tests/dp_engine_test.py

```diff
@@ -152,7 +155,13 @@ def test_aggregate_none(self):
         self.assertIsNone(
             pipeline_dp.DPEngine(None, None).aggregate(None, None, None))
 
-    def test_aggregate_report(self):
+    @patch('pipeline_dp.accumulator.create_accumulator_params')
+    def test_aggregate_report(self, mock_create_accumulator_params_function):
```

Comment:
Thanks for extending this test! In future when Reports will be added we will be able just to add checks on the content of _report_generators.

For context, AggregationReports will provide for the library user a way to understand how computations were performed (eg. which noise is added etc)

Reply:
Got it! Thanks!